docs/.vitepress/config/en.ts

```diff
@@ -127,6 +127,10 @@ export const enConfig: LocaleSpecificConfig<DefaultTheme.Config> = {
             text: 'Issues',
             link: 'https://github.com/Tresjs/tres/issues',
           },
+          {
+            text: 'Contributing',
+            link: 'https://github.com/Tresjs/tres/blob/main/CONTRIBUTING.md',
```

Comment:
@colinscz should be https://github.com/Tresjs/.github/blob/main/CONTRIBUTING.md

Reply:
addressed in de8a2a660a37e560c202e8af25258a6f95e2ec24